WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/IssueRefundViewModel.swift

```diff
@@ -297,22 +314,26 @@ extension IssueRefundViewModel {
         return Section(rows: [switchRow, detailsRow])
     }
 
-    /// If the order has fees, this returns a row with a message that refunding fees is currently
-    /// not supported.
-    ///
-    /// We will implement refunding fees within a few weeks. This is just temporary and is primarily
-    /// for setting expectations for Simple Payments users.
-    ///
-    private func createUnsupportedFeesRefundTooltipRow() -> ImageAndTitleAndTextTableViewCell.ViewModel? {
-        state.order.fees.isEmpty ? nil : ImageAndTitleAndTextTableViewCell.ViewModel(
-            title: Localization.unsupportedFeesRefund,
-            titleFontStyle: .footnote,
-            text: nil,
-            image: .infoOutlineFootnoteImage,
-            imageTintColor: .systemColor(.secondaryLabel),
-            numberOfLinesForTitle: 0,
-            isActionable: false,
-            showsSeparator: false)
+    private func createFeesSection() -> Section? {
+        guard ServiceLocator.featureFlagService.isFeatureFlagEnabled(.refundFees)
+        else {
+            return nil
+        }
+
+        guard isAnyFeeAvailableForRefund() == false else {
+            return nil
+        }
```

Comment:
Small style nit:

```suggestion
        guard ServiceLocator.featureFlagService.isFeatureFlagEnabled(.refundFees), isAnyFeeAvailableForRefund() else {
            return nil
        }
```

Reply:
Oh, great suggestion! I've fixed it here: https://github.com/woocommerce/woocommerce-ios/pull/5663/commits/68c240ce7966f7729f874dacf7c1441bdeae860a. But after removing the Fee Refunds feature flag here https://github.com/woocommerce/woocommerce-ios/pull/5663/commits/c1d20d78cd6a15fcdc1a8af5dadf30d6ccb1499f the refactor ended up not changing much, but it gave me the opportunity to fix the `isAnyFeeAvailableForRefund` boolean semantic, which was flipped using the `isEmpty` instead of `isNotEmpty` inside the fee list.